src/main/java/mmosii/bookstore/controller/BookController.java

```diff
@@ -18,40 +21,48 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "api/books")
 public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> findAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books", description = "Get a list of all available books")
+    public List<BookDto> findAll(Pageable pageable) {
+        return bookService.findAll(pageable);
     }
 
     @GetMapping("/{id}")
+    @Operation(summary = "Get a book", description = "Get a book by id, if this id exists")
     public BookDto getBookById(@PathVariable Long id) {
```

Comment:
just FYI

you may also validate such things as @PathVariable, @RequestParams etc. It looks like:
```
@PathVariable @Positive Long id
```

In order to be able to perform such validation you need to annotate your controller class with `@Validated` annotation

again, this is just FYI - apply it or not :)